CHANGELOG.txt

```diff
@@ -1,5 +1,19 @@
-14 Aug. 2014 update from version 1.0.3.2 to version  1.1
+2018 TBD version 1.2.0
```

Comment:
TBD? shouldn't that be `vbap`?

Reply:
TBD usually means "To Be Determined" and is a placeholder as I didn't know when the version would be finished/accepted.

Reply:
> In looking at this again, I think the PDDP stuff should not have been removed. I can put it back.

Ah never mind. What I removed was just pddp links to other objects, not the meta files.